src/components/cards/cardSummary/cardSummary.api.ts

```diff
@@ -0,0 +1,37 @@
+import type { IconType } from '../../icon';
+import type { ICardProps } from '../card/card';
+
+export interface ICardSummaryAction {
```

Comment:
Why narrow this to merely the label and onClick? Do we anticipate that we won't need to change/remove the button icon for example?

Reply:
Exactly, currently only the `label` will be customisable and the icon will always be displayed. We can change it later if requirements will change :)